### DISCUSSION TITLE
How to silence `$XONSH_SHOW_TRACEBACK` help text?

### DISCUSSION THREAD

**Discussion by rpdelaney** [xonsh/xonsh#4888] [Q&A] [answered]
```
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
```
I get this every time there's a shell error. I know about it and I don't need to be reminded. How can I silence this help text?

**Comment by anki-code** [comment]
You can add to your `~/.xonshrc`:
```python
$XONSH_SHOW_TRACEBACK = False
# or
$XONSH_TRACEBACK_LOGFILE = '/dev/null'
```
Example:
```python
xonsh --no-rc
raise Exception(1)
# xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
# Exception: 1

$XONSH_TRACEBACK_LOGFILE = '/dev/null'
raise Exception(1)
# Exception: 1

$XONSH_SHOW_TRACEBACK = True
raise Exception(1)
#Traceback (most recent call last):
#  File "<stdin>", line 1, in <module>
#Exception: 1
```

Here is the logic in the code:

https://github.com/xonsh/xonsh/blob/cbf23e60fba9c6479d23cff03884572a776e6f1d/xonsh/tools.py#L963-L972

**Comment by rpdelaney** [comment, marked as answer]
Thanks to @anki-code 

If `XONSH_SHOW_TRACEBACK` is manually set to `False` then the help text is not printed. My problem was that I was setting it like this:

```
XONSH_SHOW_TRACEBACK=False
```
...when I should have done this:
```
$XONSH_SHOW_TRACEBACK=False
```
